meeting_minutes/220714_SARIF_TC_62.md

```diff
@@ -0,0 +1,157 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair David)
+
+## 1.2 Introduction of participants/roll call (Co-Chair David)
+
+| First Name | Last Name | Company           | Role(s)                 |
+|:-----------|:----------|:------------------|:------------------------|
+| Aditya     | Sharad    | Microsoft         | Member                  |
+| Aharon     | Abadi     | WhiteSource       | Member |
```

Comment:
The GitHub suggestions don't behave nicely with whitespace here, unfortunately. Those 3 modified rows have broken borders now.